ai/ai-agents.rst

```diff
@@ -52,6 +51,33 @@ table.
 AI Agent Types
 --------------
 
+Define your AI agents and use them afterwards in triggers or scheduler jobs.
+An example for a trigger configuration could look like this:
+
+- Activated by: **Action**
+- Action execution: **Selective**
+- Conditions for affected objects: **Action** *is* ``created``
+- Execute changes on objects: **AI Agent** > Select your AI agent
+
+.. hint:: Depending on the agent type, you can limit the available options for
+  the AI agents. If you do so, make sure to select at least two options the
+  agent can choose from.
+
+Ticket Categorizer
+^^^^^^^^^^^^^^^^^^
+
+This AI agent is capable of assigning categories to tickets. Because there is
+no built in category field in Zammad, you first have to create a
+:doc:`custom ticket attribute </system/objects>` in which you want to track the
+ticket's category. The field type must be one of the select types.
+
+The ticket categorizer AI agent uses the title and the content of the article
```

Comment:
Actually, it's *all* articles of the ticket, and in that context ticket title will be included as well.